evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1417,3 +1418,18 @@ def test_visualize_decision_trees(fitted_tree_estimators, tmpdir):
     src = visualize_decision_tree(estimator=est_reg, filled=True, max_depth=2)
     assert src.format == 'pdf'
     assert isinstance(src, graphviz.Source)
+
+
+def test_t_sne():
```

Comment:
can you add tests changing the values of the other `graph_t_sne` and `t_sne` parameters? and seeing if those will still work as we expect?